zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/job/ActivatableJobsPushTest.java

```diff
@@ -240,33 +249,30 @@ private Long createJob(
     return jobRecord.getKey();
   }
 
-  private void assertEventOrder(final Intent... eventOrder) {
-    for (final long piKey : activeProcessInstances) {
-      final var processInstanceRecordStream = records().betweenProcessInstance(piKey);
-      assertThat(processInstanceRecordStream)
-          .extracting(Record::getIntent)
-          .containsSequence(eventOrder);
-    }
-  }
-
-  private void assertJobActivations(final int activationCount) {
-    final List<Record<JobBatchRecordValue>> batchRecord =
-        RecordingExporter.jobBatchRecords(JobBatchIntent.ACTIVATED).withType(jobType).asList();
-    assertThat(batchRecord).hasSize(activationCount);
+  private void assertEventOrder(
+      final Set<ValueType> targetValueTypes, final Intent... expectedIntents) {
+    assertThat(
+            records()
+                .onlyEvents()
+                .filter(r -> targetValueTypes.contains(r.getValueType()))
+                .skipUntil(r -> r.getIntent() == expectedIntents[0])
+                .limit(expectedIntents.length))
+        .extracting(Record::getIntent)
+        .describedAs("Expected event order does not match actual order.")
+        .containsExactly(expectedIntents);
   }
 
-  private void assertActivatedJob(final Long jobKey, final int activationCount) {
+  private void assertActivatedJobs(final Long jobKey, final int activationCount) {
```

Comment:
🔧 Let's rename this to `assertPushedActivatedJobs` or `assertActivatedJobsPushed` to better reflect what this method verifies